ReportCardParser/test/reportcardparser/ReportCardStatsTest.java

```diff
@@ -5,7 +5,8 @@
  */
 package reportcardparser;
 
-import org.junit.jupiter.api.Test;
+import static org.junit.Assert.*;
```

Comment:
The tests you have written are good, but you are missing tests for edge/error cases.